discord/voice_client.py

```diff
@@ -331,6 +333,20 @@ def _encrypt_xsalsa20_poly1305_suffix(self, header, data):
 
         return header + box.encrypt(bytes(data), nonce).ciphertext + nonce
 
+    def _encrypt_xsalsa20_poly1305_lite(self, header, data):
+        box = nacl.secret.SecretBox(bytes(self.secret_key))
+        nonce = bytearray(24)
+
+        # Upper limit of unsigned 4 byte int
+        if self.lite_nonce >= 0xffffffff:
+            self.lite_nonce = 0
+
+        nonce[:4] = struct.pack('>I', self.lite_nonce)
+        self.lite_nonce += 1
```

Comment:
This is exactly what the `checked_add` function is for.

Reply:
I don't see how adding another function for this conditional is beneficial

Reply:
The library already does this here:
https://github.com/Rapptz/discord.py/blob/c5f01eb66efb82fd1d2cf32b3658113322b37c01/discord/voice_client.py#L456